src/renderer/entities/account/model/account.ts

```diff
@@ -112,10 +112,11 @@ export const isMultishard = (account?: Account | MultisigAccount): boolean => {
   return Boolean(account.walletId);
 };
 
-export function isWalletContact(account?: Account | MultisigAccount): boolean {
+export function isWalletContact(account?: Account | MultisigAccount, wallet?: WalletDS): boolean {
   if (!account) return false;
+  const notMultishard = !wallet || wallet.type !== WalletType.MULTISHARD_PARITY_SIGNER;
```

Comment:
Better use positive variables accompanied with negative operator
```suggestion
  const isMultishard = wallet && wallet.type === WalletType.MULTISHARD_PARITY_SIGNER;
```